src/Equinor.ProCoSys.IPO.Command/InvitationCommands/CreateInvitation/CreateInvitationCommandHandler.cs

```diff
@@ -331,8 +331,8 @@ public async Task<Result<int>> Handle(CreateInvitationCommand request, Cancellat
                     participant.Organization,
                     IpoParticipantType.Person,
                     null,
-                    participant.Person.FirstName, 
-                    participant.Person.LastName,
+                    null, 
```

Comment:
Can you explain why a participant can be made with some many props set to null?

Reply:
Yes - this will rarely happen, because this code will only hit if they do not have azure Oid (which all equinor users should be getting asap). So, when making the endpoints in main api, in collaboration with Pål, we decided to just create endpoints to get persons with oids, and not emails. In the case where a person has just an email, we still need to add them to the invitation, but I do not think it is necessary to send firstname and lastname from frontend when this scenario SHOULD never happen (but probably will? until everyone has azure oid). Therefore we just set name to null, but of course set the email that has been sendt from frontend. Do you think this is incorrect? We can discuss it if you want.

Reply:
Ok if it is well discussed already (even if it is kind of odd)
I guess then IPO must have code that handle participants differently due to different properties set/not set